streamlit-python/local_test_env.yml

```diff
@@ -0,0 +1,11 @@
+# This file is used to install packages for local testing
+name: streamlit-python-testing
+channels:
+  - snowflake
+dependencies:
+  - python=3.8
+  - snowflake-snowpark-python
+  - pip:
+      - pytest
+      - streamlit>=1.28.0
```

Comment:
Is it worth calling out that this diverges from the SiS version available in Native Apps (soon to be 1.26, currently 1.22?)

Reply:
I called this out in the README, but i can add a comment here.